docs/source/contribution.rst

```diff
@@ -142,6 +142,27 @@ Please note the followings when writing the document.
   the document.
 
 
+Building Guidelines
+-------------------
+
+In order to run unit tests at the repository root, you first have to build Cython files in place by running the following command::
+
+  $ python setup.py develop
+
+Please note that when you modify ``*.pxd`` files, you must clean ``*.cpp`` and ``*.so`` files once with::
```

Comment:
I think this part could be made into a ```.. note::``` section.

For English expression, how about:

>When you modify ``*.pxd`` files,  before running ``python setup.py develop``, you must clean ``*.cpp`` and ``*.so`` files with the following command, because Cython does not automatically rebuild modified those files nicely:
```
  $ git clean -fdx
```